src/legendhpges/make_hpge.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+import json
+
+from legendmeta.jsondb import AttrsDict
+from pyg4ometry import geant4
+
+from .bege import BEGe
+from .invcoax import InvertedCoax
+from .materials import _enriched_germanium
+from .ppc import PPC
+from .registry import default_g4_registry
+from .semicoax import SemiCoax
+
+
+def make_hpge(
```

Comment:
Can you please add the possibility for the user to override the material with an optional `material` argument?

Reply:
I have two options to add this possibility. Would you prefer additionally adding `material: geant4.MaterialBase = None` then doing some checking below or just putting `**kwargs`, then use something like `material = kwargs.get("material", default_material)`? In the latter, we could also get rid of the `name` argument and replacing it with `name = kwargs.get("name", gedet_meta.name)`.

Reply:
Yes, very good idea! You could just forward `kwargs` as it is to the various HPGe sub-classes constructors